ckanext/canada/schemas/presets.yaml

```
@@ -97,6 +97,9 @@ presets:
         en: Briefing packages for Parliamentary Committee appearances for deputy heads
         fr: Documents d'information à l'intention des administrateurs généraux pour les comparutions devant un comité parlementaire
       value: parliament_committee_deputy
+    - label:
+        en: Use of Administrative Aircraft
+        fr: Utilisation des avions d'affaires
```

Does adminaircraft use a custom collection type? If so it needs a value too:
```suggestion
        fr: Utilisation des avions d'affaires
      value: adminaircraft
```